doc/faq.rst

```diff
@@ -0,0 +1,166 @@
+  configuration option;
+
+- Change the order of the sections using the :ref:`section_order
+  <rsmtool_section_order>` option;
+
+- Create your own :ref:`"custom" sections <custom_notebooks>`.
+
+|hr|
+
+.. _q_no_response_remaining:
+
+:fa:`quora`. **I get the following error:** ``No responses remaining after filtering out
+non-numeric feature values. No further analysis can be run.`` **What
+happened?** [:ref:`# <q_no_response_remaining>`]
+
+``rsmtool`` is designed to work only with numeric features. Non-numeric values
+*including* missing values are filtered out. Some of the common reasons for this
+error are:
+
+- The human score column or one of the features contains only non-numeric
+  values. You can either exclude this feature or convert it to one-hot
+  encoding.
+
+- You have features with missing values. The solution is to replace
+  missing values with zeros. Note that this applies *even* if you use the
+  ``.jsonlines`` format.  We have an open `issue
+  <https://github.com/EducationalTestingService/rsmtool/issues/480>`_ to
+  provide an option to automatically convert missing values to zeros.
+
+- You have a lot of missing feature values and none of the responses has
+  numeric features for every single feature. Inspect the
+  :ref:`*_excluded_responses<rsmtool_excluded_responses>` files in the output
+  directory to see what responses have been excluded.
+
+|hr|
+
+.. _q_multiple_models:
+
+:fa:`quora`. **Can I pass a set of learners to the** ``model`` **option in the
+configuration file or do I need a separate run for each leaner I want to try?**
+[:ref:`# <q_multiple_models>`]
+
+``rsmtool`` cannot train multiple models via a configuration file. If you wish
+to use multiple learners/models, you should use the :ref:`RSMTool API <api>`
+instead of the command line.
+
+|hr|
+
+.. _q_how_to_predict_with_model:
+
+:fa:`quora`. **Can I compute predictions on new data using a model trained
+with** ``rsmtool`` **?** [:ref:`# <q_how_to_predict_with_model>`]
+
+Yes! We have built :ref:`rsmpredict<usage_rsmpredict>` to do exactly this!
+
+|hr|
+
+.. _q_changed_feature_signs:
+
+:fa:`quora`. **Why did** ``rsmtool`` **change the sign of some features in**
+``feature.csv`` **? I thought** ``rsmtool`` **assumes a positive sign for all
+features?** [:ref:`# <q_changed_feature_signs>`]
+
+``rsmtool`` indeed assumes a default positive sign for all raw features.
+However, if you set ``select_transformations`` to ``true`` in your experiment
+configuration file,  RSMTool will automatically apply
+:ref:`transformations<select_transformations_rsmtool>` to some of the features.
+Some transformations such as ``inv`` (inverse transform) change the polarity of
+the feature.  In such cases, RSMTool takes this into account and changes the
+sign accordingly. See :ref:`note here<clever_sign_note>`.
+
+|hr|
+
+.. _q_mismatched_rsmpredict_rsmtool:
+
+:fa:`quora`. I ran ``rsmpredict`` to generate predicted scores using a model
```

Comment:
This question font is not bold

Reply:
also, suggesting to make the question shorter, something like:
rsmpredict generates a different number of results than rsmtool, on the same evaluation set.